src/scripts/main.js

```diff
@@ -1,3 +1,57 @@
 'use strict';
 
-// write code here
+const appendRow = document.querySelector('.append-row');
+const removeRow = document.querySelector('.remove-row');
+const appendColumn = document.querySelector('.append-column');
+const removeColumn = document.querySelector('.remove-column');
+const table = document.querySelector('tbody');
+const MAXITEMS = 10;
+const MINITEMS = 2;
```

Comment:
```suggestion
const MAX_ITEMS = 10;
const MIN_ITEMS = 2;
```

Reply:
Thank you for help, now its works this way.) And now i know about constants in uppercase!)